distribution/docs/src/main/resources/content/_reference/_appReferences/mg-spatial.adoc

```diff
@@ -27,7 +27,9 @@ The `gazetteer` commands provide the ability to interact with the local GeoNames
 |Usage
 
 |`update`
-|Adds new gazetteer metacards to the core catalog. Entries can be a URL or local file path to GeoNames data or a local file path to GeoJSON data (GeoJSON URLs are not supported). Currently .txt and .zip files are supported for manual GeoNames entries and .geo.json files are supported for manual GeoJSON entries. GeoNames data can be manually downloaded from http://download.geonames.org/export/dump, where the absolute path of the file would be passed as an argument to the command (ex. /Users/johndoe/Downloads/AU.zip). Entries can also be automatically downloaded from http://download.geonames.org/export/dump by passing the country code as an argument to the command (ex. AU) which will add the country as GeoNames metacards in the core catalog. The full list of country codes available can be found in http://download.geonames.org/export/dump/countryInfo.txt. Using the argument "all" will download all of the current country codes (this process may take some time). In addition to country codes, GeoNames also provides entries for cities based on their population sizes. The arguments "cities1000", "cities5000", and "cities15000" will add cities to the index that have at least 1000, 5000, or 15000 people respectively. To download a default collection of geo.json country records, execute the command `gazetteer:update "$ddf.data/countries.geo.json"`.
+|Adds new gazetteer metacards to the core catalog. Entries can be a URL or local file path to GeoNames data or a local file path to GeoJSON data (GeoJSON URLs are not supported). Currently .txt and .zip files are supported for manual GeoNames entries and .geo.json files are supported for manual GeoJSON entries. GeoNames data can be manually downloaded from http://download.geonames.org/export/dump, where the absolute path of the file would be passed as an argument to the command (ex. /Users/johndoe/Downloads/AU.zip). Entries can also be automatically downloaded from http://download.geonames.org/export/dump by passing the country code as an argument to the command (ex. AU) which will add the country as GeoNames metacards in the core catalog. The full list of country codes available can be found in http://download.geonames.org/export/dump/countryInfo.txt. Using the argument "all" will download all of the current country codes (this process may take some time). In addition to country codes, GeoNames also provides entries for cities based on their population sizes. The arguments "cities1000", "cities5000", and "cities15000" will add cities to the index that have at least 1000, 5000, or 15000 people respectively. Additionally, URLs can also be used as an argument (ex. http://download.geonames.org/export/dump/AI.zip)
```

Comment:
✏️ `... people, respectively. Alternatively, a URL can be used as the argument (e.g. http://download.geonames.org/export/dump/AI.zip).`